src/client/sandbox/event/element-editing-watcher.js

```diff
@@ -1,65 +1,65 @@
-import * as DOM from '../../utils/dom';
-import * as EventSimulator from './simulator';
 import NativeMethods from '../native-methods';
```

Comment:
We still have a huge inconsistency in naming in many places: native methods here is an instance, so it should be named `nativeMethods`. Do you mind fixing all such places in context of this PR?